hackathon_site/dashboard/frontend/src/components/inventory/ProductOverview/ProductOverview.tsx

```diff
@@ -165,13 +180,26 @@ const DetailInfoSection = ({
     );
 };
 
-const MainSection = ({ name, total, quantityAvailable, categories, img }) => {
+interface MainSectionProps {
+    name: string;
+    quantityAvailable: number;
+    quantityRemaining: number;
+    categories: (string | undefined)[];
```

Comment:
This should just be a `string[]`, make the component that passes it in filter out the undefined values